src/Microsoft.Azure.WebJobs.Host/Loggers/TraceWriterFunctionInstanceLogger.cs

```diff
@@ -25,7 +25,7 @@ public TraceWriterFunctionInstanceLogger(TraceWriter trace)
 
         public Task<string> LogFunctionStartedAsync(FunctionStartedMessage message, CancellationToken cancellationToken)
         {
-            string traceMessage = string.Format(CultureInfo.InvariantCulture, "Executing: '{0}' - Reason: '{1}'", message.Function.ShortName, message.FormatReason());
+            string traceMessage = string.Format(CultureInfo.InvariantCulture, "Executing '{0}' (Reason='{1}', Id={2})", message.Function.ShortName, message.FormatReason(), message.FunctionInstanceId);
```

Comment:
I was trying to find the 'Reason' (specifically, the blob name) that triggered a failing function. I had the invocation id that I was looking for, but when you're combing through logs of hundreds of functions/minute, all the logs are interleaved and it's really hard to figure out which 'Reason' goes with the 'Id'. This just removes that issue.